src/sentry/tasks/beacon.py

```diff
@@ -121,6 +122,7 @@ def send_beacon():
     # which is the same as False
     anonymous = options.get("beacon.anonymous") is not False
     event_categories_count = get_category_event_count_24h()
+    byte_to_gigabyte = 1024**-3
 
     payload = {
```

Comment:
Do we have any tests that verify that we send the payload we expect? Would have caught the `-3` bug above if we did.